File: app/src/main/java/com/fluffies/utils/helper/MediaLoader.java
package com.fluffies.utils.helper;


import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.fluffies.R;
import com.yanzhenjie.album.AlbumFile;
import com.yanzhenjie.album.AlbumLoader;

public class MediaLoader implements AlbumLoader {

    @Override
    public void load(ImageView imageView, AlbumFile albumFile) {
        load(imageView, albumFile.getPath());
    }

    @Override
    public void load(ImageView imageView, String url) {
        Glide.with(imageView.getContext())
                .load(url)
                .error(R.drawable.loader)
                .into(imageView);
    }
}

File: app/src/main/java/com/fluffies/base/MediaLoader.kt
package com.fluffies.base

import android.widget.ImageView
import com.bumptech.glide.Glide
import com.fluffies.R

import com.yanzhenjie.album.AlbumFile
import com.yanzhenjie.album.AlbumLoader

class MediaLoader : AlbumLoader {
override fun load(imageView: ImageView, albumFile: AlbumFile) {
load(imageView, albumFile.path)
}

override fun load(imageView: ImageView, url: String) {
Glide.with(imageView.context)
.load(url)
.error(R.drawable.place_holder)
.into(imageView)
}
}